styles/tool-bar.less

```diff
@@ -61,8 +62,7 @@
   }
 
   &.tool-bar-horizontal {
-    .tool-bar-btn,
-    .tool-bar-spacer {
+    .tool-bar-btn, .tool-bar-spacer {
```

Comment:
Is this a compiling issue?

Personally, I prefer separate selectors to be on separate lines (which is a style consistently used in this file so far) as it clearly distinguishes descender selectors from separate selectors. I.e. the difference between these two:

```css
.tool-bar-btn .tool-bar-spacer { } /* style .tool-bar-spacer if it is a descendent of .tool-bar-button */
.tool-bar-btn, . tool-bar-spacer { } /* style .tool-bar-spacer and .tool-bar-button */
```

Why did you decide to change these (and not others, e.g. line 30 and 31)?

Reply:
The standard has them in the same line:
https://www.w3schools.com/cssref/css_selectors.asp

I didn't go through all the lines. If consistency matters we can do that. Personally, I don't care.

Reply:
> The standard has them in the same line:
https://www.w3schools.com/cssref/css_selectors.asp

First, standards are not style guides, and second a big reason why that standard (which it is not, it is more akin to documentation) has them on a single line is because it is in a table.

I had hoped the reasoning for my preference would make sense to you (i.e. being able to distinguish to very similar but vastly different CSS selectors), but if not I don't mind putting all selectors on a single line.


> I didn't go through all the lines. If consistency matters, we can do that for all the lines. Personally, I don't care.

I don't necessarily care about consistency. What I will say is that as a reviewer I do care that all changes in a Pull Request work towards the same goal. Because, again, is this a compiling issue?

I'm sorry if I'm being too nit-picky here :sweat_smile: Sure, for this small PR such small changes don't really matter, but I hope that you can see that in a larger PR these kinds of changes add up. I won't block this PR for these changes, but I hope that you see my point and  that you can learn from it in the future :smiley: